index.html

```diff
@@ -30,18 +36,25 @@ <h1 id="form-header">
         <li>
           <label for="vorname">Vorname*</label>
           <input
+            required
             id="vorname"
             type="text"
             placeholder="zb. Frederike oder Klaus"
           />
         </li>
         <li>
-          <label for="vorname">Nachname*</label>
-          <input id="nachname" type="text" placeholder="zb. Mustermann" />
+          <label for="nachname">Nachname*</label>
+          <input
+            required
+            id="nachname"
+            type="text"
```

Comment:
`type="text"` is default and could be ommited. But it's not wrong to leave it here.